src/sections/what-is-service-mesh/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { MDXRenderer } from "gatsby-plugin-mdx";
```

Comment:
Why import this?

Reply:
It's useless, thought of using it for the image earlier 🤦🏻‍♂️😂